cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -141,6 +144,11 @@ runQueryCmd cmd =
     QueryUTxO' consensusModeParams qFilter networkId mOutFile ->
       runQueryUTxO consensusModeParams qFilter networkId mOutFile
 
+queryErrorToShelleyQueryCmdError :: QueryError ShelleyQueryCmdError -> ShelleyQueryCmdError
```

Comment:
Why not add an additional constructor to `ShelleyQueryCmdError`? E.g `ShelleyQueryRelatedCmdError QueryError`